src/components/Inventory/InventoryTable.tsx

```diff
@@ -9,6 +9,9 @@ import { useDeviceInfo } from '@terraware/web-components/utils';
 import { APP_PATHS } from 'src/constants';
 import TextField from '@terraware/web-components/components/Textfield/Textfield';
 import InventoryCellRenderer from './TableCellRenderer';
```

Comment:
unrelated to change but might be good to rename this file to InventoryCellRenderer. The function name inside is InventoryCellRenderer.. and imported here as such as well.